### DISCUSSION TITLE
I'v seen an attached png track in an mp4 file - Do you know in what application could be a png attached onto a video as an overlay?

### DISCUSSION THREAD

**Discussion by GotoConsol** [mifi/lossless-cut#2244] [Ideas]
My basie Q is:

Is it possible in any app (Lossless Cut or other) to attach a png with transparent areas
As a track on top of the main video track
In mkv and/or mp4 and/or formats
So you would easily add texts, logos, etc onto the top a video
?

My Q based on the following attached png track in a mp4 video - seen in Lossless Cut Track Panel

The actual video 'Def Leppard - Viva! Hysteria (Full Concert HD)' is from YouTube
If you download it you can check the png track

![Lossless_Cut   001 2](https://github.com/user-attachments/assets/b38de985-aaab-44f3-a9e6-daf58b7c43fa)

**Comment by mifi** [comment]
As far as i know that is not possible. It would require multi layered video rendering support in video codecs
